### PR TITLE
update readme with ssl/tls configuration

### PR DESCRIPTION
**Pull request by karakakou0**
## Description

> A few sentences describing the overall goals of the pull request's commits.

## Related PRs

> List related PRs against other branches:

| name         | link                                                            |
| ------------ | --------------------------------------------------------------- |
| This is a PR | [link](https://github.com/Ca-Cybersecurity-Robiot/robiot/pulls) |

## Remains

- [ ] One sub task
- [ ] Another things to complete task

## Screenshots / Videos

> Share visual changes if necessary

## Checklist before asking for reviews

- [ ] own review
- [ ] `npm run test` passes on my machine
- [ ] is my code well documented

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,133 @@
 ![Build, Quality, Test](https://github.com/Ca-Cybersecurity-Robiot/robiot/workflows/Build,%20Quality,%20Test/badge.svg)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e7d04395ccee48d5bcf0ed850cfc1078)](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot?utm_source=github.com&utm_medium=referral&utm_content=Ca-Cybersecurity-Robiot/robiot&utm_campaign=Badge_Grade_Settings)
 
+## SSL/TLS Sécurisation du lien entre robiot et API
+
+l'objectif de ce chapitre est de présenter une méthode d'authentification basée sur les certificats SSL/TLS et la configuration tomcat.
+Cette méthode requiert au minimum 3 certificats :
+ - un certificat pour l'autorité de certification, ce certificat peut être généré manuellement 
+ ou bien vous pouvez acheter les signatures nécessaire auprès d'une autorité de certification 
+ - Un certificat pour le serveur (à faire signer par l'autorité de certification)
```
Comment (MaolmeoX) [inline README.md:11]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md [hunk 2]
```diff
@@ -2,6 +2,133 @@
...
+ - Un certificat pour le client (à faire signer par l'autorité de certification)
```
Comment (MaolmeoX) [inline README.md:12]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md (hunk 1 above)
Comment (MaolmeoX) [inline README.md:11]: Codacy found an issue: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md [hunk 3]
```diff
@@ -2,6 +2,133 @@
 ![Build, Quality, Test](https://github.com/Ca-Cybersecurity-Robiot/robiot/workflows/Build,%20Quality,%20Test/badge.svg)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e7d04395ccee48d5bcf0ed850cfc1078)](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot?utm_source=github.com&utm_medium=referral&utm_content=Ca-Cybersecurity-Robiot/robiot&utm_campaign=Badge_Grade_Settings)
 
+## SSL/TLS Sécurisation du lien entre robiot et API
+
+l'objectif de ce chapitre est de présenter une méthode d'authentification basée sur les certificats SSL/TLS et la configuration tomcat.
+Cette méthode requiert au minimum 3 certificats :
+ - un certificat pour l'autorité de certification, ce certificat peut être généré manuellement
```
Comment (MaolmeoX) [inline README.md:9]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

Comment (MaolmeoX) [inline README.md:9]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md (hunk 1 above)
Comment (MaolmeoX) [inline README.md:11]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md [hunk 4]
```diff
@@ -2,6 +2,133 @@
 ![Build, Quality, Test](https://github.com/Ca-Cybersecurity-Robiot/robiot/workflows/Build,%20Quality,%20Test/badge.svg)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e7d04395ccee48d5bcf0ed850cfc1078)](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot?utm_source=github.com&utm_medium=referral&utm_content=Ca-Cybersecurity-Robiot/robiot&utm_campaign=Badge_Grade_Settings)
 
+## SSL/TLS Sécurisation du lien entre robiot et API
+
+l'objectif de ce chapitre est de présenter une méthode d'authentification basée sur les certificats SSL/TLS et la configuration tomcat.
+Cette méthode requiert au minimum 3 certificats :
+ - un certificat pour l'autorité de certification, ce certificat peut être généré manuellement 
+ ou bien vous pouvez acheter les signatures nécessaire auprès d'une autorité de certification
```
Comment (MaolmeoX) [inline README.md:10]: Codacy found an issue: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md (hunk 2 above)
Comment (MaolmeoX) [inline README.md:12]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md [hunk 5]
```diff
@@ -2,6 +2,132 @@
 ![Build, Quality, Test](https://github.com/Ca-Cybersecurity-Robiot/robiot/workflows/Build,%20Quality,%20Test/badge.svg)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e7d04395ccee48d5bcf0ed850cfc1078)](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot?utm_source=github.com&utm_medium=referral&utm_content=Ca-Cybersecurity-Robiot/robiot&utm_campaign=Badge_Grade_Settings)
 
+## SSL/TLS Sécurisation du lien entre robiot et API
+
+l'objectif de ce chapitre est de présenter une méthode d'authentification basée sur les certificats SSL/TLS et la configuration tomcat.
+Cette méthode requiert au minimum 3 certificats :
+- un certificat pour l'autorité de certification, ce certificat peut être généré manuellement ou bien vous pouvez acheter les signatures nécessaire auprès d'une autorité de certification 
+- Un certificat pour le serveur (à faire signer par l'autorité de certification)
+- Un certificat pour le client (à faire signer par l'autorité de certification)
```
Comment (MaolmeoX) [inline README.md:11]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md [hunk 6]
```diff
@@ -2,6 +2,132 @@
 ![Build, Quality, Test](https://github.com/Ca-Cybersecurity-Robiot/robiot/workflows/Build,%20Quality,%20Test/badge.svg)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e7d04395ccee48d5bcf0ed850cfc1078)](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot?utm_source=github.com&utm_medium=referral&utm_content=Ca-Cybersecurity-Robiot/robiot&utm_campaign=Badge_Grade_Settings)
 
+## SSL/TLS Sécurisation du lien entre robiot et API
+
+l'objectif de ce chapitre est de présenter une méthode d'authentification basée sur les certificats SSL/TLS et la configuration tomcat.
+Cette méthode requiert au minimum 3 certificats :
+- un certificat pour l'autorité de certification, ce certificat peut être généré manuellement ou bien vous pouvez acheter les signatures nécessaire auprès d'une autorité de certification 
+- Un certificat pour le serveur (à faire signer par l'autorité de certification)
```
Comment (MaolmeoX) [inline README.md:10]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md [hunk 7]
```diff
@@ -2,6 +2,132 @@
 ![Build, Quality, Test](https://github.com/Ca-Cybersecurity-Robiot/robiot/workflows/Build,%20Quality,%20Test/badge.svg)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e7d04395ccee48d5bcf0ed850cfc1078)](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot?utm_source=github.com&utm_medium=referral&utm_content=Ca-Cybersecurity-Robiot/robiot&utm_campaign=Badge_Grade_Settings)
 
+## SSL/TLS Sécurisation du lien entre robiot et API
+
+l'objectif de ce chapitre est de présenter une méthode d'authentification basée sur les certificats SSL/TLS et la configuration tomcat.
+Cette méthode requiert au minimum 3 certificats :
+- un certificat pour l'autorité de certification, ce certificat peut être généré manuellement ou bien vous pouvez acheter les signatures nécessaire auprès d'une autorité de certification
```
Comment (MaolmeoX) [inline README.md:9]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md [hunk 8]
```diff
@@ -2,6 +2,132 @@
 ![Build, Quality, Test](https://github.com/Ca-Cybersecurity-Robiot/robiot/workflows/Build,%20Quality,%20Test/badge.svg)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e7d04395ccee48d5bcf0ed850cfc1078)](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot?utm_source=github.com&utm_medium=referral&utm_content=Ca-Cybersecurity-Robiot/robiot&utm_campaign=Badge_Grade_Settings)
 
+## SSL/TLS Sécurisation du lien entre robiot et API
+
+l'objectif de ce chapitre est de présenter une méthode d'authentification basée sur les certificats SSL/TLS et la configuration tomcat.
```
Comment (MaolmeoX) [inline README.md]:
```suggestion
L'objectif de ce chapitre est de présenter une méthode d'authentification basée sur les certificats SSL/TLS et la configuration de Tomcat.
```

File: README.md [hunk 9]
```diff
@@ -2,6 +2,132 @@
...
+Cette méthode requiert au minimum 3 certificats :
+ - Un certificat pour l'autorité de certification, ce certificat peut être généré manuellement ou bien vous pouvez acheter les signatures nécessaire auprès d'une autorité de certification
```
Comment (MaolmeoX) [inline README.md]:
```suggestion
 - Un certificat pour l'autorité de certification, ce certificat peut être généré manuellement ou bien vous pouvez acheter les signatures nécessaires auprès d'une autorité de certification 
```

File: README.md [hunk 10]
```diff
@@ -2,6 +2,132 @@
...
+openssl req -new -key CAkey.key -out CAReq.csr -config /etc/ssl/openssl.cnf 
+
+# enfin nous signons nous même le certificat avec la commande suivante
+openssl x509 -req -days 365 -in CAReq.csr -signkey CAKey.key -out CA.crt
+```
+
+### 2 -  Création du certificat pour le serveur
+
+```shell script
+# Création d'une clef privée pour notre serveur
```
Comment (MaolmeoX) [inline README.md]:
```suggestion
# Création d'une clé privée pour notre serveur
```

File: README.md [hunk 11]
```diff
@@ -2,6 +2,132 @@
...
+## SSL/TLS Sécurisation du lien entre robiot et API
+
+l'objectif de ce chapitre est de présenter une méthode d'authentification basée sur les certificats SSL/TLS et la configuration tomcat.
+Cette méthode requiert au minimum 3 certificats :
+ - Un certificat pour l'autorité de certification, ce certificat peut être généré manuellement ou bien vous pouvez acheter les signatures nécessaire auprès d'une autorité de certification 
+ - Un certificat pour le serveur (à faire signer par l'autorité de certification)
+ - Un certificat pour le client (à faire signer par l'autorité de certification)
+ 
+Après l'obtention des certificats l'objectif est de mettre à jour la configuration de tomcat afin activer l'authentification 
+d'un client par certificat. Enfin nous ajoutons des contraintes de securités afin de limiter les url et méthode HTTP disponible.
```
Comment (MaolmeoX) [inline README.md]:
```suggestion
Après l'obtention des certificats l'objectif est de mettre à jour la configuration de Tomcat afin activer l'authentification 
d'un client par certificat. Enfin nous ajoutons des contraintes de sécurités afin de limiter les URL et méthode HTTP disponible.
```

File: README.md [hunk 12]
```diff
@@ -2,6 +2,132 @@
...
+```
+
+### 2 -  Création du certificat pour le serveur
+
+```shell script
+# Création d'une clef privée pour notre serveur
+openssl genrsa -out ServerKey.key 1024
+
+# Création du CSR basé sur la clé. 
+# Après la saisie de cette commande il faut répondre à quelque question
```
Comment (MaolmeoX) [inline README.md]:
```suggestion
# Après la saisie de cette commande il faut répondre à quelques questions
```

File: README.md [hunk 13]
```diff
@@ -2,6 +2,132 @@
...
+```
+
+### 3 -  Création du certificat pour le client
+
+```shell script
+# Création d'une clef privée pour notre client
+openssl genrsa -out ClientKey.key 1024
+
+# Création du CSR basé sur la clé. 
+# Après la saisie de cette commande il faut répondre à quelque question
```
Comment (MaolmeoX) [inline README.md]:
```suggestion
# Après la saisie de cette commande il faut répondre à quelques questions
```

File: README.md [hunk 14]
```diff
@@ -2,6 +2,132 @@
...
+openssl req -new -key ClientKey.key -out CleintReq.csr -config /etc/ssl/openssl.cnf 
+
+# enfin nous signons nous même le certificat avec celui de l'autorité de certification
```
Comment (MaolmeoX) [inline README.md]:
```suggestion
# Enfin nous signons nous même le certificat avec celui de l'autorité de certification 
```

File: README.md [hunk 15]
```diff
@@ -2,6 +2,132 @@
...
+
+```shell script
+# Création d'une clef privée pour notre CA
+openssl genrsa -out CAkey.key 1024
+
+# Création du CSR basé sur la clé. 
+# Après la saisie de cette commande il faut répondre à quelque question
+openssl req -new -key CAkey.key -out CAReq.csr -config /etc/ssl/openssl.cnf 
+
+# enfin nous signons nous même le certificat avec la commande suivante
```
Comment (MaolmeoX) [inline README.md]:
```suggestion
# Enfin nous signons nous même le certificat avec la commande suivante
```

File: README.md [hunk 16]
```diff
@@ -2,6 +2,132 @@
...
+openssl req -new -key ClientKey.key -out CleintReq.csr -config /etc/ssl/openssl.cnf 
+
+# enfin nous signons nous même le certificat avec celui de l'autorité de certification 
+openssl x509 -req -days 365 -CA CA.crt -CAkey CAKey.key -CAcreateserial -in CleintReq.csr -out Client.crt
+```
+
+### 4 -  Convertion d'un certificat du format crt ver p12
+
+```shell script
+# Nous convertissons le certificat client au format p12 pour effectuer les test depuis notre navigateur
```
Comment (MaolmeoX) [inline README.md]:
```suggestion
# Convertir le certificat client au format p12 pour effectuer les tests depuis notre navigateur
```

File: README.md [hunk 17]
```diff
@@ -2,6 +2,132 @@
...
+openssl pkcs12 -export -in Cleint.crt -inkey ClientKey.key -chain -CAfile CA.crt -out ClientCert.p12
+
+```
+
+### 5 -  Création du truststore pour le serveur tomcat
+
+```shell script
+# Nous importons le certificat de l'autorité de certification au sein du truststore
```
Comment (MaolmeoX) [inline README.md]:
```suggestion
# Importer le certificat de l'autorité de certification au sein du truststore
```

File: README.md [hunk 18]
```diff
@@ -2,6 +2,132 @@
...
+keytool -import -alias CertAuth -keystore caCerts.jks -file CA.crt
+```
+
+### 5 -  Création du keystore pour le serveur tomcat
+
+```shell script
+# Nous pouvons utiliser cette commande pour importer tous les certificats au quels nous souhaitons autoriser l'acces
```
Comment (MaolmeoX) [inline README.md]:
```suggestion
# Permet d'importer les certificats dont nous souhaiton autoriser l'accès.
```

File: README.md [hunk 19]
```diff
@@ -2,6 +2,132 @@
...
+           truststoreFile="/opt/tomcat/apache-tomcat-9.0.37/certificat/CA/caCerts.jks" truststorePass="password"
+	   keystoreFile="/opt/tomcat/apache-tomcat-9.0.37/certificat/tomcat.keystore" keystorePass="password"
+           clientAuth="false" sslProtocol="TLS"
+           />
+```
+
+### 7 -  Mettre à jour les utilisateurs dans le fichier tomcat-user.xml
+
+```shell script
+  # Nous créons un utilisateur avec les informations du certificat généré
```
Comment (MaolmeoX) [inline README.md]:
```suggestion
  # Création d'un utilisateur avec les informations du certificat généré
```

**Review by MaolmeoX — CHANGES_REQUESTED** [review]

File: README.md [hunk 20]
```diff
@@ -2,6 +2,132 @@
 ![Build, Quality, Test](https://github.com/Ca-Cybersecurity-Robiot/robiot/workflows/Build,%20Quality,%20Test/badge.svg)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e7d04395ccee48d5bcf0ed850cfc1078)](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot?utm_source=github.com&utm_medium=referral&utm_content=Ca-Cybersecurity-Robiot/robiot&utm_campaign=Badge_Grade_Settings)
 
+## SSL/TLS Sécurisation du lien entre robiot et API
+
+L'objectif de ce chapitre est de présenter une méthode d'authentification basée sur les certificats SSL/TLS et la configuration de Tomcat.
+Cette méthode requiert au minimum 3 certificats :
+ - Un certificat pour l'autorité de certification, ce certificat peut être généré manuellement ou bien vous pouvez acheter les signatures nécessaires auprès d'une autorité de certification 
+ - Un certificat pour le serveur (à faire signer par l'autorité de certification)
+ - Un certificat pour le client (à faire signer par l'autorité de certification)
```
Comment (MaolmeoX) [inline README.md:11]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md [hunk 21]
```diff
@@ -2,6 +2,132 @@
 ![Build, Quality, Test](https://github.com/Ca-Cybersecurity-Robiot/robiot/workflows/Build,%20Quality,%20Test/badge.svg)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e7d04395ccee48d5bcf0ed850cfc1078)](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot?utm_source=github.com&utm_medium=referral&utm_content=Ca-Cybersecurity-Robiot/robiot&utm_campaign=Badge_Grade_Settings)
 
+## SSL/TLS Sécurisation du lien entre robiot et API
+
+L'objectif de ce chapitre est de présenter une méthode d'authentification basée sur les certificats SSL/TLS et la configuration de Tomcat.
+Cette méthode requiert au minimum 3 certificats :
+ - Un certificat pour l'autorité de certification, ce certificat peut être généré manuellement ou bien vous pouvez acheter les signatures nécessaires auprès d'une autorité de certification 
+ - Un certificat pour le serveur (à faire signer par l'autorité de certification)
```
Comment (MaolmeoX) [inline README.md:10]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md [hunk 22]
```diff
@@ -2,6 +2,132 @@
 ![Build, Quality, Test](https://github.com/Ca-Cybersecurity-Robiot/robiot/workflows/Build,%20Quality,%20Test/badge.svg)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e7d04395ccee48d5bcf0ed850cfc1078)](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot?utm_source=github.com&utm_medium=referral&utm_content=Ca-Cybersecurity-Robiot/robiot&utm_campaign=Badge_Grade_Settings)
 
+## SSL/TLS Sécurisation du lien entre robiot et API
+
+L'objectif de ce chapitre est de présenter une méthode d'authentification basée sur les certificats SSL/TLS et la configuration de Tomcat.
+Cette méthode requiert au minimum 3 certificats :
+ - Un certificat pour l'autorité de certification, ce certificat peut être généré manuellement ou bien vous pouvez acheter les signatures nécessaires auprès d'une autorité de certification
```
Comment (MaolmeoX) [inline README.md:9]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

Comment (MaolmeoX) [inline README.md:9]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md (hunk 20 above)
Comment (MaolmeoX) [inline README.md:11]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md (hunk 21 above)
Comment (MaolmeoX) [inline README.md:10]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md [hunk 23]
```diff
@@ -79,3 +205,4 @@ auto-fix :
 ``
 npm run lint -- --fix
 ``
+
```
Comment (foch01) [inline README.md]: Pourquoi une ligne est ajouter à la fin ?

File: README.md [hunk 24]
```diff
@@ -2,6 +2,132 @@
 ![Build, Quality, Test](https://github.com/Ca-Cybersecurity-Robiot/robiot/workflows/Build,%20Quality,%20Test/badge.svg)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e7d04395ccee48d5bcf0ed850cfc1078)](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot?utm_source=github.com&utm_medium=referral&utm_content=Ca-Cybersecurity-Robiot/robiot&utm_campaign=Badge_Grade_Settings)
 
+## SSL/TLS Sécurisation du lien entre robiot et API
+
+L'objectif de ce chapitre est de présenter une méthode d'authentification basée sur les certificats SSL/TLS et la configuration tomcat.
+Cette méthode requiert au minimum 3 certificats :
+ - Un certificat pour l'autorité de certification, ce certificat peut être généré manuellement ou bien vous pouvez acheter les signatures nécessaires auprès d'une autorité de certification
+ - Un certificat pour le serveur (à faire signer par l'autorité de certification)
```
Comment (MaolmeoX) [inline README.md:10]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md [hunk 25]
```diff
@@ -2,6 +2,132 @@
 ![Build, Quality, Test](https://github.com/Ca-Cybersecurity-Robiot/robiot/workflows/Build,%20Quality,%20Test/badge.svg)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e7d04395ccee48d5bcf0ed850cfc1078)](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot?utm_source=github.com&utm_medium=referral&utm_content=Ca-Cybersecurity-Robiot/robiot&utm_campaign=Badge_Grade_Settings)
 
+## SSL/TLS Sécurisation du lien entre robiot et API
+
+L'objectif de ce chapitre est de présenter une méthode d'authentification basée sur les certificats SSL/TLS et la configuration tomcat.
+Cette méthode requiert au minimum 3 certificats :
+ - Un certificat pour l'autorité de certification, ce certificat peut être généré manuellement ou bien vous pouvez acheter les signatures nécessaires auprès d'une autorité de certification
```
Comment (MaolmeoX) [inline README.md:9]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md (hunk 24 above)
Comment (MaolmeoX) [inline README.md:10]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md [hunk 26]
```diff
@@ -2,6 +2,132 @@
...
+ - Un certificat pour le serveur (à faire signer par l'autorité de certification)
+ - Un certificat pour le client (à faire signer par l'autorité de certification)
```
Comment (MaolmeoX) [inline README.md:11]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

Comment (MaolmeoX) [inline README.md:11]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md (hunk 25 above)
Comment (MaolmeoX) [inline README.md:9]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941841)

File: README.md (hunk 23 above)
Comment (karakakou0) [reply README.md]: désoler grand gourou je retire la ligne, ... travail terminé

Comment (foch01) [reply README.md]: Merci 😇

**Review by MaolmeoX — APPROVED** [review]

**Review by foch01 — APPROVED** [review]

**Pull request merged**